debian/control

```diff
@@ -3,7 +3,7 @@ Section: graphics
 Priority: optional
 Maintainer: Canon Inc. <sup-debian@list.canon.co.jp>
 Build-Depends: debhelper (>= 4.0.0), libcupsys2-dev | libcups2-dev,
-               libxml2-dev, libtiff4-dev, automake, autoconf,
+               libxml2-dev, libtiff5-dev, automake, autoconf,
```

Comment:
You're right. I didn't do that because I did not see any `Provides` line for libtiff4, but there's indeed one for libtiff5, so we should use it. Updated